client-side-js/testLibrary.js

```diff
@@ -10,7 +10,13 @@ async function initOPA(pageObjectConfig, browserInstance) {
                         pageConfig[pageKey] = new window.fe_bridge[className](options)
                     })
                 })
+
                 sap.ui.test.Opa5.createPageObjects(pageConfig)
+                // use the same timouts and intervals that wdi5 uses
+                sap.ui.test.Opa.extendConfig({
+                    timeout: new Date(window.wdi5.waitForUI5Options.timeout).getSeconds(), // convert milliseconds to seconds
```

Comment:
can `window.wdi5.waitForUI5Options.timeout` be `undefined`?

Reply:
In the [injectUi5](https://github.com/ui5-community/wdi5/blob/main/client-side-js/injectUI5.js#L23) script we make sure this property can not be undefined.